### PR TITLE
Use an object for estimated_duration in tasks.update

### PR DESCRIPTION
**Pull request by psrpinto**
Follow-up to https://github.com/teamleadercrm/api/pull/299

https://github.com/teamleadercrm/core/pull/8311

## Note
Since we're not ready to release these endpoints yet, we decided to not include the `tasks.apib` in the `src/apiary.apib` file. That way we can iterate on the docs through multiple PRs. Once we're ready to release, we will include `tasks.apib` in `src/apiary.apib`.

### CODE REVIEW

**Review by WouterSioen — APPROVED** [review]
yes please

**Review by r1dotl — APPROVED** [review]

**Review by mrtus — COMMENTED** [review]
Missing a build file though.

**Comment by psrpinto** [comment]
@mrtus We willingly removed the `tasks.apib` from `src/apiary.apib` so we could iterate on the docs. Once we're ready to release we'll add it back.

 I keep forgetting to mentioning this in the PR description.

File: src/08-tasks/tasks.apib [hunk 1]
```diff
@@ -105,7 +105,11 @@ Update a tasks.
         + description (string, optional)
         + due_at: `2016-02-04T16:44:33+00:00` (string, optional)
         + work_type_id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string, optional)
-        + estimated_duration: 3600 (number, optional) - In seconds
+        + estimated_duration (object, optional)
+            + unit: `s` (enum[string], required)
+                + Members
+                    + s - Seconds
+            + value: `60` (number, required)
```
Comment (tiagobastos) [inline src/08-tasks/tasks.apib:112]: Noob Q: Does apiary allows a way to state that we require a positive number? 🤔

**Review by tiagobastos — APPROVED** [review]

File: src/08-tasks/tasks.apib (hunk 1 above)
Comment (WouterSioen) [reply src/08-tasks/tasks.apib:112]: you can add comments if necessary

**Pull request merged**